packages/core/src/MachineNode.ts

```diff
@@ -251,7 +259,19 @@ export class MachineNode<
    */
   public get initialState(): State<TContext, TEvent, TStateSchema, TTypestate> {
     this._init();
-    const nextState = getInitialState(this);
+    const nextState = resolveMicroTransition(this, [], undefined, undefined);
+    return macrostep(nextState, null as any, this);
+  }
+
+  public getInitialState(service?: ActorRef<TEvent>) {
```

Comment:
what's the story around this? do you plan to remove `.initialState`? how the behavior of this one is going to differ when `service` is given to it and when it is not?

Reply:
For initial states with spawned actors, `machine.initialState` will have actors in context that are not started and have no parent. Same with `machine.getInitialState()`.

With `machine.getInitialState(service)`, the actors will not be started (until the service starts them) but will have a parent. That's the distinction.